### DISCUSSION TITLE
Invalid Copilot token

### DISCUSSION THREAD

**Discussion by MasterBrian99** [community/community#126246] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

few hours ago i lost access to github copilot even though i have a active subsciption.
in vs code no matter how many ways tried it always getting the same issue.
![2024-05-28_20-33](https://github.com/community/community/assets/37585474/97888a7d-6431-42d2-82ba-2952f733edbb)
```bash
2024-05-28 20:25:02.013 [info] [auth] Invalid copilot token: missing token: 403 
2024-05-28 20:25:02.168 [error] [auth] Extension activation failed: "Contact Support. You are currently logged in as MasterBrian99."
2024-05-28 20:25:28.209 [info] [auth] Invalid copilot token: missing token: 403 
```

same with intellij idea.
![Screenshot_2024-05-28-20-34-02_3360x1080](https://github.com/community/community/assets/37585474/7fc9a8ea-8cf3-458e-9280-aa81b2da6269)

i tried with cli.same issue.i have an active subscription.
![image](https://github.com/community/community/assets/37585474/d47b0558-55c4-4540-a152-0139de093cce)

here is my last receipt.

![2024-05-28_20-38](https://github.com/community/community/assets/37585474/6457ebef-a2d6-4e2b-83c1-2d04dbad10ed)

i don't know where to go from here.

**Comment by davevad93** [comment, marked as answer]
Hi @MasterBrian99 ,

I guess you've already tried to logging out/in from `Copilot` and reinstalling the extension in your `IDE` so if even that hasn't worked, you should open a ticket [here](https://support.github.com/contact/account) describing your issue.

**Reply by MasterBrian99** [reply]
still working with the support team.thank you for the info

**Reply by tblakex01** [reply]
The same exact issue and error message happened to me, as well on the same day no less. After looking up the error I found out my account was randomly flagged with no explanation and my profile was hidden from view and I can no longer use Copilot. I'm still waiting for someone to reply to my support ticket. Did you get any explanation on how to fix the issue?
